package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "daedalus",
   "productName": "Daedalus",
-  "version": "2.2.0",
+  "version": "2.1.0",
```

Comment:
@daniloprates this change is for testing purposes only right? We need to set it to 2.2.0 before merging 🙏

Reply:
Yes, we need it order to test the automatic download of `2.2.0`.